Dockerfile.worker

```diff
@@ -0,0 +1,32 @@
+# https://hub.docker.com/_/microsoft-dotnet-core
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-bionic AS build
+RUN dotnet --version
+WORKDIR /source
+
+RUN apt-get update && apt-get install -y unzip
+# RUN ln -s /usr/lib/x86_64-linux-gnu/libssl.so.1.1 /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
+# RUN ln -s /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
```

Comment:
Can these two lines be deleted? Are they needed for reference for some reason?

Reply:
Removed in f112bac.